src/core/data/index.ts

```diff
@@ -15,24 +15,30 @@ export enum PersonRole {
   EMPLOYER = 'EMPLOYER'
 }
 
-export interface Person {
+export interface Person<DateType = Date> {
   firstName: string
   lastName: string
   ssid: string
   role: PersonRole
+  isBlind: boolean
+  dateOfBirth: DateType
```

Comment:
You'll need to address these changes in a data migration, if you can't make these two fields optional. Otherwise a user that has data in local storage but does not have these fields will have errors.

See: 
https://github.com/ustaxes/UsTaxes/blob/2c3ec7c2ec9d1049d38e927b2894933795748a08/src/redux/store.ts#L79

https://github.com/ustaxes/UsTaxes/blob/2c3ec7c2ec9d1049d38e927b2894933795748a08/src/redux/migration.ts#L10

Reply:
Should I add in another function to be run after`migrateEachYear` or do I add into that function?

Reply:
I think it would be a new function. The idea with migrations is that each time the state type is changed we add a function which updates the users state to match the expected type. 

So if those functions are called in sequence (chronologically) then hopefully no matter how old someone’s data is it will be corrected to the current type. 

I’m sure we will need a more sophisticated solution soon.